README.md

```diff
@@ -143,6 +143,9 @@ Options:
   --force-missing-dependencies                Force importing and syncing resources that
                                               could be potential dependencies to the
                                               requested resources. [sync only]
+  --create-global-downtime                    Schedule a downtime for monitors synced by
+                                              datadog-sync-cli. Scheduled downtime should be
+                                              manually removed after failover is complete.
```

Comment:
Instead of "Scheduled downtime should be manually removed after failover is complete", maybe "Scheduled downtime is meant to be removed during failover when customer determines monitors have enough telemetry to trigger appropriately"?